lib/tutorials/en_US/getting-started.md

```diff
@@ -45,89 +51,96 @@ Now that we have a server we should add one or two routes so that it actually do
 
-To install inert run this command at the command line: `npm install --save inert` This will download inert and add it to your `package.json`, which documents which packages are installed.
+To install [inert](https://github.com/hapijs/inert) run this command at the command line: `npm install --save inert` This will download [inert](https://github.com/hapijs/inert) and add it to your `package.json`, which documents which packages are installed.
 
-Add the following to your `server.js` file:
+Update the `init` function in your `server.js` file:
 
 ``` javascript
-server.register(require('inert'), (err) => {
+const init = async () => {
 
-    if (err) {
-        throw err;
-    }
+    await server.register(require('inert'));
 
     server.route({
         method: 'GET',
         path: '/hello',
-        handler: function (request, reply) {
-            reply.file('./public/hello.html');
+        handler: (request, h) => {
+
+            return h.file('./public/hello.html');
         }
     });
-});
 
+    await server.start();
+    console.log(`Server running at: ${server.info.uri}`);
+};
 
 ```
 
-The `server.register()` command above adds the inert plugin to your Hapi application. If something goes wrong, we want to know, so we've passed in an anonymous function which if invoked will receive `err` and throw that error. This callback function is required when registering plugins.
+The `server.register()` command above adds the [inert](https://github.com/hapijs/inert) plugin to your hapi application.
 
-The `server.route`() command registers the `/hello` route, which tells your server to accept GET requests to `/hello` and reply with the contents of the `hello.html` file. We've put the routing callback function inside of registering inert because we need to insure that inert is registered _before_ we use it to render the static page. It is generally wise to run code that depends on a plugin within the callback that registers that plugin so that you can be absolutely sure that plugin exists when your code runs.
+The `server.route`() command registers the `/hello` route, which tells your server to accept GET requests to `/hello` and reply with the contents of the `hello.html` file. We've put the route registration after registering the `inert` plugin. It is generally wise to run code that depends on a plugin after the plugin is registered so that you can be absolutely sure that the plugin exists when your code runs.
 
-Start up your server with `npm start` and go to `http://localhost:3000/hello` in your browser. Oh no! We're getting a 404 error because we never created a `hello.html` file. You need to create the missing file to get rid of this error.
+Start up your server with `npm start` and go to [`http://localhost:3000/hello`](http://localhost:3000/hello) in your browser. Oh no! We're getting an error because we never created a `hello.html` file. You need to create the missing file to get rid of this error.
```

Comment:
Sorry noticed another thing unrelated to your PR but we should fix. `npm start` won't work because we didn't add a start script. This should be `node server.js`